selfdrive/ui/qt/widgets/controls.h

```diff
@@ -101,7 +101,7 @@ class ToggleControl : public AbstractControl {
     QObject::connect(&toggle, &Toggle::stateChanged, this, &ToggleControl::toggleFlipped);
   }
 
-  void setEnabled(bool enabled) { toggle.setEnabled(enabled); }
+  void setEnabled(bool enabled) { toggle.setEnabled(enabled); toggle.repaint(); }
```

Comment:
generally, prefer update to repaint. update queues up a repaint whereas repaint forces one
```suggestion
  void setEnabled(bool enabled) { toggle.setEnabled(enabled); toggle.update(); }
```

Reply:
Ah I forgot, thanks!